cli/Makefile

```diff
@@ -118,9 +78,6 @@ publish-turbo: clean build
 
 	npm config set --location=project "//registry.npmjs.org/:_authToken" $(NPM_TOKEN)
 
-	# Publishes the native npm modules.
-	goreleaser release --rm-dist -f combined-shim.yml $(SKIP_PUBLISH)
-
```

Comment:
I don't think we can get rid of this. It is executing some code that prepares the various platform packages.

Reply:
brought back all the Makefile and `*.yml` and goreleaser stuff